src/core/libraries/kernel/memory.cpp

```diff
@@ -318,6 +318,10 @@ s32 PS4_SYSV_ABI sceKernelIsStack(void* addr, void** start, void** end) {
     return memory->IsStack(std::bit_cast<VAddr>(addr), start, end);
 }
 
+u32 PS4_SYSV_ABI sceKernelIsAddressSanitizerEnabled() {
+    return ORBIS_OK;
```

Comment:
why that should be always be true? i know it was a default behavior but still doesn't feels right

Reply:
either way i think at least we need to add a log here , to at least know that it gets called

Reply:
added 👍